india_stocks_api/brokers/angel.py

```
@@ -321,143 +283,114 @@ def modify_gtt(self, id: int, instrument: Equity | Future | Option,
+        jwt_token = self._require_auth()
+
         payload = {
             "id": id,
             "symboltoken": token_info["token"],
             "exchange": token_info["exchange"]
         }
-        
+
         return cancel_gtt_rule(payload, jwt_token)
 
     def get_gtt_list(self, status: list = ["FOR_SETTLEMENT", "CANCELLED", "TRIGGERED"]) -> list:
```

Ruff (0.15.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)